packages/camera_driver/include/hardware_test_camera/hardware_test_camera.py

```diff
@@ -0,0 +1,42 @@
+import rospy
+
+from dt_duckiebot_hardware_tests import HardwareTest, HardwareTestJsonParamType
+
+
+class HardwareTestCamera(HardwareTest):
+    def __init__(self) -> None:
+        super().__init__()
+
+    def test_id(self) -> str:
+        return "Camera"
+
+    def test_description_preparation(self) -> str:
+        return self.html_util_ul(
+            ["Place the Duckiebot within your reach, as you will need to move it during the test."]
+        )
+
+    def test_description_expectation(self) -> str:
+        return self.html_util_ul(["A live camera stream should be shown below."])
+
+    def cb_run_test(self, _):
+        rospy.loginfo(f"[{self.test_id()}] Test service called.")
+
+        instructions = self.html_util_ul(
```

Comment:
Note @jasonhu5 - I think the instructions should stay in consistent places as a few of these change format.  I personally prefer that the "preparation" contains setup instructions and the "expectation" contains all other test instructions, so will stick with that in the following comments.